packages/cdk-ecs-alb/lib/index.ts

```diff
@@ -0,0 +1,156 @@
+import * as cdk from "@aws-cdk/core";
+import * as ecs from "@aws-cdk/aws-ecs";
+import * as ecr from "@aws-cdk/aws-ecr";
+import * as ec2 from "@aws-cdk/aws-ec2";
+import * as elbv2 from "@aws-cdk/aws-elasticloadbalancingv2";
+import * as iam from "@aws-cdk/aws-iam";
+import * as ecspattern from "@aws-cdk/aws-ecs-patterns";
+// import * as logs from "@aws-cdk/aws-logs";
+
+export interface CdkEcsAlbProps {
+    clusterName: string;
+    appName: string;
+    vpcId: string;
+    securityGroupId: string;
+    repositoryName: string;
+    stage: string;
+    tag?: string;
+}
+
+export class CdkEcsAlb extends cdk.Construct {
+    constructor(scope: cdk.Construct, id: string, props: CdkEcsAlbProps) {
+        super(scope, id);
+
+        let loadBalancer: elbv2.ApplicationLoadBalancer | elbv2.NetworkLoadBalancer;
+        let listener: elbv2.ApplicationListener | elbv2.NetworkListener | void;
+
+        var taskRole = new iam.Role(this, "EcsTaskRole", {
+            assumedBy: new iam.ServicePrincipal("ecs-tasks.amazonaws.com")
+        });
+
+        taskRole.addToPolicy(new iam.PolicyStatement({
+            actions: [
+                "ssm:GetParameters",
+                "ssm:PutParameter",
+                "ssm:GetParameter",
+                "secretsmanager:GetSecretValue",
+                "kms:Decrypt"
+            ],
+            resources: [
+                "*"
```

Comment:
We need to further restrict this so that the task can only pull a decrypt the ssm parameters that it needs.